defaults/test_default.yaml

```diff
@@ -170,3 +170,4 @@ data_volume_disk_num: 0
 data_volume_disk_type: ''
 data_volume_disk_size: 0
 data_volume_disk_iops: 0 # depend on type iops could be 100-16000 for io2|io3 and 3000-16000 for gp3
+nosqlbench_image: 'scylla0dkropachev/nosqlbench:latest'
```

Comment:
need to move it to our own repo: `scylladb/hydra-loaders`

Reply:
Moved there